### DISCUSSION TITLE
live servers problem???

### DISCUSSION THREAD

**Discussion by t0ny54** [DNSCrypt/dnscrypt-proxy#2184] [Q&A] [answered]
I have Forwarder DNS Server on https://euroresolver.herokuapp.com/dns-query
Google chrome, adguard home & nebulo can use that server, but dnscrypt-proxy respond that server not live...

DNS Stamps 
sdns://AgMAAAAAAAAAAAAaZXVyb3Jlc29sdmVyLmhlcm9rdWFwcC5jb20KL2Rucy1xdWVyeQ

Log on heroku for app connect:

> 2022-08-24T10:08:08.951168+00:00 heroku[router]: at=info method=GET path="/dns-query?dns=1lcBIAABAAAAAAAABHBsYXkKZ29vZ2xlYXBpcwNjb20AAAEAAQ" host=euroresolver.herokuapp.com request_id=2a95135b-1e40-4aa5-a200-c040e13222bc fwd="xxx.xxx.xxx.xxx" dyno=web.1 connect=0ms service=64ms status=200 bytes=206 protocol=https

> 2022-08-24T10:11:34.698904+00:00 heroku[router]: at=info method=POST path="/dns-query" host=euroresolver.herokuapp.com request_id=54dcaab7-8993-4f66-80bf-81a7b28583f8 fwd="xxx.xxx.xxx.xxx" dyno=web.1 connect=0ms service=61ms status=200 bytes=263 protocol=https

For dnscrypt-proxy is blank log on heroku

How do I find out where the problem is?

Thank You

**Comment by lifenjoiner** [comment, marked as answer]
It works, by my test:
1. Set `static` with your stamp, and turn off `sources`;
2. Specify `resolver_names` with 'myserver';

Turning on log, I got:
```
[2022-08-24 19:43:52] [WARNING] [myserver] does not support HTTP/2
[2022-08-24 19:43:52] [INFO] [myserver] TLS version: 303 - Protocol: http/1.x - Cipher suite: 49199
[2022-08-24 19:43:52] [NOTICE] [myserver] OK (DoH) - rtt: 231ms
[2022-08-24 19:43:52] [NOTICE] Server with the lowest initial latency: myserver (rtt: 231ms)
[2022-08-24 19:43:52] [NOTICE] dnscrypt-proxy is ready - live servers: 1
[2022-08-24 19:44:13]   127.0.0.1       www.google.com  A       PASS    245ms   myserver
```

For more details: [Configuration-Sources](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Configuration-Sources)

**Reply by t0ny54** [reply]
Oh it is my fault. Because i use 
> tls_cipher_suite = [4867, 52393] on dnscrypt-proxy.toml 

I can't connect that server...
Thank you for your test
